setup-dev.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
please move this file to the `scripts/` folder instead of the root & update the target and symlink paths to adjust

also a new line at the end